packages/realm-server/tests/helpers/index.ts

```diff
@@ -220,11 +224,11 @@ export async function runTestRealmServer({
   dir,
   fileSystem,
   realmURL,
-  permissions,
   virtualNetwork,
   queue,
   dbAdapter,
   matrixConfig,
+  permissions = { '*': ['read', 'write'] },
```

Comment:
we should probably default in our tests to whatever the default permissions are for realms--i don't think world writable is the default realm permission.

Reply:
Agree. Will change it.

Reply:
Fixed in https://github.com/cardstack/boxel/pull/1306/commits/d64e0edfd4276b29310d604cb63963966248f7b2 but the world readable remains because this is the current default, but as mentioned in my other comment I think we should discuss what are new defaults should be. Will create a ticket for that.